### PR TITLE
SV-485 get standard options list

### PR DESCRIPTION
**Pull request by ben1stone**

### CODE REVIEW

File: IntegrationTests/Outer API Tests For Standards.postman_collection.json [hunk 1]
```diff
@@ -79,6 +79,106 @@
 			},
 			"response": []
 		},
+		{
+			"name": "Courses - Get List - None",
```
Comment (DevenKShah) [inline IntegrationTests/Outer API Tests For Standards.postman_collection.json:83]: Might I suggest to create a separate collection for options.

**Review by DevenKShah — CHANGES_REQUESTED** [review]
Need a few things fixed. Also unit test for conversion of GetStandardOptionsListItem => GetStandardOptionsItem might help.

**Comment by ben1stone** [comment]
Reverted commit 611df02 as these tests have been added to their own collection

**Comment by CodeWolf86** [comment]
I've changed the base to be SV-372 as that will be our next branch to deploy, It contains the new Assessors Outer Api changes with new detail endpoint, so can include the options work as well.

This should coincide with SV-484 and SV-482 of courses api as our next deliverable

**Review by CodeWolf86 — APPROVED** [review]

**Comment by ben1stone** [comment]
Closed - a new branch has been created and merged

**Pull request closed without merging**